lib/Pod/Simple/Wiki/Confluence.pm

```diff
@@ -174,7 +174,10 @@ sub _end_L {
 
     # Handle links that are parsed as Pod links.
     if ( defined $link_section ) {
-        $link_target = $link_section;
+        # Other than for L<Link Text>, having a link section defined means
+        # that this is a page anchor link.
+        $link_target = ( $link_attrs->{raw} eq "$link_section" ) ? '' : '#';
```

Comment:
I'm having some trouble with the failing tests...the cause of the failures is that in Perl 5.10, 5.12, 5.14, the `$link_attrs` hash doesn't contain the "raw" entry, and that seems to be the only way to differentiate `L<Doc Link>` from `L<"PageAnchor">` or `L</"PageAnchor">`.

Any suggestions?